tools/src/icon4pytools/py2fgen/wrappers/diffusion.py

```diff
@@ -35,7 +35,7 @@
 )
 from icon4py.model.common import dimension as dims
 from icon4py.model.common.constants import DEFAULT_PHYSICS_DYNAMICS_TIMESTEP_RATIO
-from icon4py.model.common.grid.horizontal import CellParams, EdgeParams
+from icon4py.model.common.grid.geometry import CellParams, EdgeParams
```

Comment:
```suggestion
from icon4py.model.common.grid import geometry
```

(and then have to fix the lines where it's used)